### DISCUSSION TITLE
How long should I wait before I can update via Homebrew?

### DISCUSSION THREAD

**Discussion by nathancashion** [jgm/pandoc#9751] [Q&A] [answered]
I just saw [the email](https://groups.google.com/g/pandoc-announce/c/LrCIO9fWisw/m/qBmK0sTWAQAJ) from earlier today announcing the release of Pandoc 3.2.

I tried updating via Homebrew but got the warning:
`pandoc 3.1.13 already installed`

How long does it take for the Homebrew packages to be updated to the latest release?

**Comment by alerque** [comment, marked as answer]
Homebrew is entirely managed by a group of volunteers and accepts contributions via GitHub. Asking upstream projects for timelines on that sort of thing won't help much.

In this case it looks like somebody opened [a pull request for Pandoc 3.2](https://github.com/Homebrew/homebrew-core/pull/171468) before you opened this. Searching the Homebrew issues and PRs would be the place to get info about things that are in progress.

My experience is Homebrew is pretty fast on updates for most projects unless there are blocking reasons across an ecosystem (like needing a new version of a dependency that affects a lot of other packages). Anything from a few hours to a few days is common.

**Reply by nathancashion** [reply]
THanks, appreciate the info!

**Comment by iandol** [comment]
Homebrew is usually super fast, the change was commited yesterday ~5pm. The main problem for delayed pandoc releases is actually the dependencies like pandoc-crossref, homebrew will delay the release of pandoc unitl the dependencies are also ready to go. That means even if you don't use a dependency you'll need to wait for its update. Thankfully, the dependecy authors are attentive and fast too!
